test/fw/ptl/utils/pbs_logutils.py

```diff
@@ -369,26 +389,30 @@ def match_msg(self, lines, msg, allmatch=False, regexp=False,
                           specified time
         :param endtime: If set ignore matches that occur after
                         specified time
-         :param syslog: If True, checks for syslog format of date.
+        :param syslog: If True, checks for syslog format of date.
                        Defaults to False.
-        :type str
+        :type bool
         """
         linecount = 0
         ret = []
         if lines:
             for l in lines:
                 if syslog:
-                    dt_str = l[:15]
+                    if starttime is not None:
```

Comment:
There's no reason for these int conversions to be inside the if syslog block right? there's a "is starttime is not None" block below as well, please move the int conversion to that block. Same comment for endtime.

Reply:
We want to int starttime and endtime only in case of syslog=true, otherwise the value can be float for microsecond logging. Therefore I have not added it to the below  "is starttime is not None" block.